pdns/pdnsutil.cc

```diff
@@ -598,6 +598,27 @@ int checkZone(DNSSECKeeper &dk, UeberBackend &B, const DNSName& zone, const vect
     }
   }
 
+  std::map<std::string, std::vector<std::string>> metadatas;
+  if (!B.getAllDomainMetadata(zone, metadatas)) {
+    cout << "[Error] Unable to retrieve metadata for zone " << zone << endl;
+    numerrors++;
+  }
+
+  for (const auto &metaData : metadatas) {
+    set<string> seen;
+    set<string> messaged;
+
+    for (const auto &value : metaData.second) {
+      if (seen.count(value) <= 0) {
+        seen.insert(value);
+      } else if (messaged.count(value) <= 0) {
+        cout << "[Error] Found duplicate metadata key value pair for zone " << zone << " with key " << metaData.first << " and value " << value << endl;
```

Comment:
`[Error] Found duplicate metadata key value pair for zone 1.example.com with key PUBLISH-CDS and value `


```suggestion
        cout << "[Error] Found duplicate metadata key value pair for zone " << zone << " with key '" << metaData.first << "' and value '" << value << "'" << endl;
```